README.md

```diff
@@ -14,7 +14,8 @@ For a detailed solution anatomy, please refer to [Solution Anatomy](./Docs/solut
 
 ## Pre-requisites and recommendations
 
-- Azure Subscription - an Azure subscription to deploy the components
+- Azure Subscription - an Azure subscription to deploy the components. You may need a global administrator role to elevate access to management groups in the subscription's tenant. Detail instrucution is available [here](https://learn.microsoft.com/en-us/azure/role-based-access-control/elevate-access-global-admin#elevate-access-for-a-global-administrator).
```

Comment:
@sjyang18 - I'm going to re-work this a bit.  Giving it some thought, we should avoid suggesting that anything needs to be deployed at the tenant root management group level if we can (this includes the policy definitions). I will see if I can expand upon your suggested changes.